zookeeper-server/src/main/java/org/apache/zookeeper/ClientCnxn.java

```diff
@@ -160,7 +157,7 @@
 
     private long sessionId;
 
-    private byte sessionPasswd[] = new byte[16];
+    private byte[] sessionPasswd;
```

Comment:
Initialized version not used?

Reply:
Right. Always be initialized in constructor.